gnosis/gp-ui
# transaction view

# Summary

Closes #851, #852 

# To Test
1. make search that returns successful result.
ex: 
![image](https://user-images.githubusercontent.com/25584298/145438455-60533503-beba-4c06-bb9b-c0ea98ba984f.png)
2. follow the link under Order id to get order details
![image](https://user-images.githubusercontent.com/25584298/145438786-0e779f70-72bf-405a-a670-242211ee8ac2.png)
3. follow the link under Transaction Hash to see the TransactionDetails page

# Background

*Optional: Give background information for changes you've made, that might be difficult to explain via comments*
